README.md

```diff
@@ -6,6 +6,10 @@
 
 - go v1.13.x
 - operator-sdk v0.19.2
+- docker
+- [opm](https://github.com/operator-framework/operator-registry/releases)
+- [podman](https://podman.io/)
```

Comment:
i think that, for now, docker is the build engine tool that we are using.

Reply:
podman is required for opm - indexing image is created by opm as podman image

Reply:
ok, then I would say to default everything to podman.

Reply:
ok

Reply:
Removed the optional to docker because the make scripts use docker and the docker images aren't available to podman to be pushed on quay